lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -213,7 +213,7 @@ fn connect_rdp_inner(
     )?;
     // Generate a random 8-digit PIN for our smartcard.
     let mut rng = rand_chacha::ChaCha20Rng::from_entropy();
-    let pin = format!("{:08}", rng.gen_range(0..99999999));
+    let pin = format!("{:08}", rng.gen_range(0i32..=99999999i32));
```

Comment:
What's up with the explicit type?

Reply:
Rust 1.59 (which I'm running locally but we don't use for builds yet) complains about it.